elixir/apps/web/lib/web/live/settings/account.ex

```diff
@@ -71,24 +72,61 @@ defmodule Web.Settings.Account do
       <:title>
         Danger zone
       </:title>
+      <:action>
+        <.button_with_confirmation
+          id="delete_account"
+          style="danger"
+          icon="hero-trash-solid"
+          on_confirm="delete_account"
+          disabled={@delete_requested}
+        >
+          <:dialog_title>Confirm Account Deletion</:dialog_title>
+          <:dialog_content>
+            This Account will be scheduled for complete deletion.<br /><br />
+            Are you sure you want to delete your Account?
```

Comment:
(nit) I'm not sure we've been capitalizing `Account` since it's not a "unique" term in our system. (these rules always seem somewhat arbitrary)

Reply:
Sounds good.  I default to capitalizing all of the things that are tables in our DB, but I'm fine with not capitalizing this.